### PR TITLE
Stop linting files in src/external

### LINKED ISSUE #1342: Disable linter for files in /external
Alternatively, add a way to omit files/folders from being linted.

### PR DESCRIPTION
**Pull request by zwrawr**
Adds `.eslingignore` and `.stylelintignore` files to stop unwantd liniting. 
Closes #1342

### CODE REVIEW

File: src/com/fonts.less [hunk 1]
```diff
@@ -8,7 +8,7 @@
 html, button, input, ._font1 {
 	/* Lato - Standard text font (Google Fonts) */
 	font-family: "Roboto", "Helvetica Neue", "Arial Nova", "Segoe UI", "Ubuntu Light", sans-serif;
-/*	font-family: "Lato", "Helvetica Neue", "Roboto", "Arial Nova", "Segoe UI", "Ubuntu Light", sans-serif;*/
+	/*	font-family: "Lato", "Helvetica Neue", "Roboto", "Arial Nova", "Segoe UI", "Ubuntu Light", sans-serif;*/
```
Comment (povrazor) [inline src/com/fonts.less]: why?

Comment (zwrawr) [reply src/com/fonts.less]: The Less linter wants the comment to be indented correctly within it's scope.

Comment (povrazor) [reply src/com/fonts.less]: Okay.

**Comment by Noojuno** [comment]
Looking forward to this :p Currently sitting at 18 errors while playing with redux

**Pull request merged**